schema/eurostat_berd_data.yaml

```diff
@@ -0,0 +1,8 @@
+API: Eurostat API
+endpoint: http://ec.europa.eu/eurostat/wdds/rest/data/v2.1/json/en/rd_e_gerdreg?sinceTimePeriod=2012&geoLevel=nuts2&precision=1&sectperf=BES&unit=EUR_HAB&unit=MIO_EUR
+schema:
+  geo: string

```

Comment:
Since this is a bit ambiguous (in ES we have geopoint being an object `{lat, long}`) I'd turn this into:

```yaml
geo:
  type: string
  description: name of the location # Cornwall and Isles of Scilly
```

Reply:
btw are these a known region type? LAD, NUTS?

Reply:
Okay, that makes sense to do. I'll make those changes. Yes, they're NUTS 2 regions (I''ll include that too).

Reply:
awesome thanks!